query-node/schema.graphql

```diff
@@ -54,6 +49,8 @@ type Membership @entity {
 
   "The type of subscription the member has purchased if any."
   subscription: BigInt
+
+  videos: [Video!] @derivedFrom(field: "ownerMember")
```

Comment:
Shouldn't this be `channels`? I don't see a use case in which we would want videos of a given member, we always fetch videos in context of a channel

Reply:
Yes, it should. Fixed